### ISSUE TITLE
Unable to discard untracked file

### ISSUE THREAD

**Issue by sindresorhus** [git-up/GitUp#101]
In a git repo, if I create a file, and then try to click the "Discard File Changes" button, I get an error message. Would be nice if I could quickly remove unstaged files.

![screen shot 2016-03-31 at 16 35 26](https://cloud.githubusercontent.com/assets/170270/14171818/23a8f956-f75f-11e5-94b4-ddfec4ce9e2b.png)

<img width="532" alt="screen shot 2016-03-31 at 16 35 43" src="https://cloud.githubusercontent.com/assets/170270/14171821/286dcb9c-f75f-11e5-909e-df469d69c4a7.png">

**Comment by swisspol** [comment]
What exact version of GitUp are you using? Does the problem go away if you relaunch GitUp? Can you copy paste here the output of `git status` here?

**Comment by sindresorhus** [comment]
GitUp 1.0.4 (1012). git 2.8.0 (but was the same with earlier git versions too). Issue persist between restarts and can be reproduced in any repo.

```
~/dev/refined-github master*
❯ git status
On branch master
Your branch is up-to-date with 'origin/master'.
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	foo

nothing added to commit but untracked files present (use "git add" to track)
```

**Comment by swisspol** [comment]
GitUp doesn't use Git tool at all but libgit2. I can repro, looks like a bug with the Simple Commit view. The Advanced Commit view doesn't have this problem.

**Comment by swisspol** [comment]
Fixed in 40526f150dc9e64a05f6bec3a6da746ee7caa68c.
